{"uid":"50c26d5ff99e7888","name":"Unsuccessful Login","fullName":"Login functionality for GuviZenClass portal: Unsuccessful Login","historyId":"07c7955c108e91d7b183d93221352f2c","time":{"start":1753451689914,"stop":1753451716303,"duration":26389},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x0x7ff6321de935+77845]\n\tGetHandleVerifier [0x0x7ff6321de990+77936]\n\t(No symbol) [0x0x7ff631f99cda]\n\t(No symbol) [0x0x7ff631ff06aa]\n\t(No symbol) [0x0x7ff631ff095c]\n\t(No symbol) [0x0x7ff632043d07]\n\t(No symbol) [0x0x7ff63201890f]\n\t(No symbol) [0x0x7ff632040b07]\n\t(No symbol) [0x0x7ff6320186a3]\n\t(No symbol) [0x0x7ff631fe1791]\n\t(No symbol) [0x0x7ff631fe2523]\n\tGetHandleVerifier [0x0x7ff6324b684d+3059501]\n\tGetHandleVerifier [0x0x7ff6324b0c0d+3035885]\n\tGetHandleVerifier [0x0x7ff6324d0400+3164896]\n\tGetHandleVerifier [0x0x7ff6321f8c3e+185118]\n\tGetHandleVerifier [0x0x7ff63220054f+216111]\n\tGetHandleVerifier [0x0x7ff6321e72e4+113092]\n\tGetHandleVerifier [0x0x7ff6321e7499+113529]\n\tGetHandleVerifier [0x0x7ff6321ce298+10616]\n\tBaseThreadInitThunk [0x0x7ff976d1e8d7+23]\n\tRtlUserThreadStart [0x0x7ff977f3c34c+44]\n\n","statusTrace":"  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 58, in step_error_message\n    raise e\n  File \"features\\steps\\login_steps.py\", line 52, in step_error_message\n    error = WebDriverWait(context.driver, 5).until(\n        EC.visibility_of_element_located((By.XPATH, \"//div[contains(text(), 'Invalid')]\"))\n    )\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x0x7ff6321de935+77845]\n\tGetHandleVerifier [0x0x7ff6321de990+77936]\n\t(No symbol) [0x0x7ff631f99cda]\n\t(No symbol) [0x0x7ff631ff06aa]\n\t(No symbol) [0x0x7ff631ff095c]\n\t(No symbol) [0x0x7ff632043d07]\n\t(No symbol) [0x0x7ff63201890f]\n\t(No symbol) [0x0x7ff632040b07]\n\t(No symbol) [0x0x7ff6320186a3]\n\t(No symbol) [0x0x7ff631fe1791]\n\t(No symbol) [0x0x7ff631fe2523]\n\tGetHandleVerifier [0x0x7ff6324b684d+3059501]\n\tGetHandleVerifier [0x0x7ff6324b0c0d+3035885]\n\tGetHandleVerifier [0x0x7ff6324d0400+3164896]\n\tGetHandleVerifier [0x0x7ff6321f8c3e+185118]\n\tGetHandleVerifier [0x0x7ff63220054f+216111]\n\tGetHandleVerifier [0x0x7ff6321e72e4+113092]\n\tGetHandleVerifier [0x0x7ff6321e7499+113529]\n\tGetHandleVerifier [0x0x7ff6321ce298+10616]\n\tBaseThreadInitThunk [0x0x7ff976d1e8d7+23]\n\tRtlUserThreadStart [0x0x7ff977f3c34c+44]\n\n","statusTrace":"  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 58, in step_error_message\n    raise e\n  File \"features\\steps\\login_steps.py\", line 52, in step_error_message\n    error = WebDriverWait(context.driver, 5).until(\n        EC.visibility_of_element_located((By.XPATH, \"//div[contains(text(), 'Invalid')]\"))\n    )\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Given I open the login page","time":{"start":1753451693341,"stop":1753451700397,"duration":7056},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0},{"name":"When I enter invalid credentials","time":{"start":1753451700397,"stop":1753451700814,"duration":417},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0},{"name":"And I click the login button","time":{"start":1753451700814,"stop":1753451704004,"duration":3190},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0},{"name":"Then I should see an error message","time":{"start":1753451704004,"stop":1753451714016,"duration":10012},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x0x7ff6321de935+77845]\n\tGetHandleVerifier [0x0x7ff6321de990+77936]\n\t(No symbol) [0x0x7ff631f99cda]\n\t(No symbol) [0x0x7ff631ff06aa]\n\t(No symbol) [0x0x7ff631ff095c]\n\t(No symbol) [0x0x7ff632043d07]\n\t(No symbol) [0x0x7ff63201890f]\n\t(No symbol) [0x0x7ff632040b07]\n\t(No symbol) [0x0x7ff6320186a3]\n\t(No symbol) [0x0x7ff631fe1791]\n\t(No symbol) [0x0x7ff631fe2523]\n\tGetHandleVerifier [0x0x7ff6324b684d+3059501]\n\tGetHandleVerifier [0x0x7ff6324b0c0d+3035885]\n\tGetHandleVerifier [0x0x7ff6324d0400+3164896]\n\tGetHandleVerifier [0x0x7ff6321f8c3e+185118]\n\tGetHandleVerifier [0x0x7ff63220054f+216111]\n\tGetHandleVerifier [0x0x7ff6321e72e4+113092]\n\tGetHandleVerifier [0x0x7ff6321e7499+113529]\n\tGetHandleVerifier [0x0x7ff6321ce298+10616]\n\tBaseThreadInitThunk [0x0x7ff976d1e8d7+23]\n\tRtlUserThreadStart [0x0x7ff977f3c34c+44]\n\n","statusTrace":"  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 58, in step_error_message\n    raise e\n  File \"features\\steps\\login_steps.py\", line 52, in step_error_message\n    error = WebDriverWait(context.driver, 5).until(\n        EC.visibility_of_element_located((By.XPATH, \"//div[contains(text(), 'Invalid')]\"))\n    )\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"stepsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":4},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Login functionality for GuviZenClass portal"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"50c26d5ff99e7888.json","parameterValues":[]}